openstudiocore/src/bimserver/ProjectImportation.hpp

```diff
@@ -0,0 +1,92 @@
+/**********************************************************************
+*  Copyright (c) 2008-2015, Alliance for Sustainable Energy.
+*  All rights reserved.
+*
+*  This library is free software; you can redistribute it and/or
+*  modify it under the terms of the GNU Lesser General Public
+*  License as published by the Free Software Foundation; either
+*  version 2.1 of the License, or (at your option) any later version.
+*
+*  This library is distributed in the hope that it will be useful,
+*  but WITHOUT ANY WARRANTY; without even the implied warranty of
+*  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+*  Lesser General Public License for more details.
+*
+*  You should have received a copy of the GNU Lesser General Public
+*  License along with this library; if not, write to the Free Software
+*  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+**********************************************************************/
+
+#ifndef BIMSERVER_PROJECTIMPORTATION_HPP
+#define BIMSERVER_PROJECTIMPORTATION_HPP
+
+#include "BIMserverAPI.hpp"
+#include "BIMserverConnection.hpp"
+
+#include "../model/Model.hpp"
+
+#include <QDialog>
+#include <QListWidget>
+#include <QStatusBar>
+#include <QSettings>
+#include <QEventLoop>
+
+namespace openstudio {
+namespace bimserver {
+
+		/// This shows a input dialog to gather project id for import
+		class BIMSERVER_API ProjectImportation: public QDialog
+		{
+			Q_OBJECT
+
+		public:
+
+			/// Default constructor
+			ProjectImportation(QWidget *parent);
+
+			/// Start importing IFC workflow
+			boost::optional<model::Model> run();
+
```

Comment:
@macumber Hi Dan, it might not be so straightforward to output warnings or errors in our workflow. We could certainly log the warnings provided by VersionTranslator. 

In a normal workflow, our OSMSerializer on BIMserver could output something to the BIMserver console or log files. But it might not be easy to transfer that message back to OpenStudio. Can we leave this out for now and discuss about it later?